documentation/modules/ROOT/pages/basics.adoc

```diff
@@ -7,7 +7,7 @@
 [.console-input]
 [source,bash,subs="+macros,+attributes"]
 ----
-mvn "io.quarkus:quarkus-maven-plugin:{quarkus-version}:create" \
+mvn "io.quarkus:quarkus-maven-plugin:create" \
```

Comment:
Is the version not needed ? Nice, didn't knew that

Reply:
Is not needed as it will create automatically the project with latest released version. I preferred to do this instead of using the Quarkus CLI to have some control over the  version used.